lib/facter.rb

```diff
@@ -346,7 +368,7 @@ def search_path
     # Gets a hash mapping fact names to their values
     # The hash contains core facts, legacy facts, custom facts and external facts (all facts that can be resolved).
     #
-    # @return [FactCollection] hash with fact names and values
+    # @return a hash with fact names and values
```

Comment:
returns

Reply:
the "@return" form is everywhere, I didn't wanted to break the pattern.